packages/hash/frontend/src/pages/[account-slug]/types/entity-type/[entity-type-id].page/shared/entity-type-context.tsx

```diff
@@ -0,0 +1,14 @@
+import { EntityType } from "@blockprotocol/type-system";
+import { createContext, useContext } from "react";
+
+export const EntityTypeContext = createContext<null | EntityType>(null);
```

Comment:
(optional)

It'd be nice to export `EntityTypeProvider = EntityTypeContext.Provider` instead. If `EntityTypeContext` remains a local var, we are protected from exposing this implementation detail to the rest of the codebase. This can simplify potential future refactoring.